src/kagglehub/datasets.py

```diff
@@ -60,3 +72,65 @@ def dataset_upload(
     )
 
     create_dataset_or_version(h, tokens, version_notes)
+
+
+def load_dataset(
+    adapter: KaggleDatasetAdapter,
```

Comment:
Do we want a default value or do we prefer always having the user explicitly set what they want?

Maybe explicit is better, the user will know what they will get.

Reply:
Yeah, because we now have 2 to start (instead of just the one HF one), I preferred explicit.